src/pages/myGeotab/guides/codeBase/usingInJava.tsx

```diff
@@ -14,14 +16,14 @@ const mavenDependency: ReactNode = (
             data over the wire. It includes tools to assist authenticating against Geotab's servers, automatically serializing/deserializing
             JSON, and providing definitions for Checkmate object classes.</p>
         <InformationalBox>
-            <p>Quick start in <a href="../../api/clients">{/*TODO: fix link*/}API Clients</a></p>
+            <p>Quick start in <Link to="/myGeotab/apiClients">API Clients</Link></p>
```

Comment:
@kmets89 The current site has hash links for the Javascript & Java pages so that the link goes to the specific section of the API Clients page.
```suggestion
            <p>Quick start in <HashLink to="/myGeotab/apiClients#java">API Clients</HashLink></p>
```